File: src/test/scala/NYTAkka/FileIteratorTest.scala
package NYTAkka

import org.scalatest.FlatSpec

/**
 * Created by anie on 3/14/2015.
 */
class FileIteratorTest extends FlatSpec {

  "A file iterator" should "iterate over values" in {

    val file = FileIterator("E:\\Allen\\NYTFuture\\NYT", true)
    while (file.hasNext) {
//      println(file.nextLine())
      file.next()
    }

  }

  "A print header" should "print" in {
//    Start.printHeader(Some(Array(0,3,4,1,2)))
  }
}

File: src/test/java/FolderReadingNIO/FileIteratorTest.java
package FolderReadingNIO;

import org.junit.Test;

import static org.junit.Assert.*;

public class FileIteratorTest {

    @Test
    public void testNextLine() throws Exception {
        FileIterator fi = new FileIterator("E:\\Allen\\NYTFuture\\NYT");
        while (fi.hasNext())
            fi.nextLine();
    }
}